doc/source/whatsnew/v0.25.0.rst

```diff
@@ -258,6 +258,7 @@ Performance Improvements
 Bug Fixes
 ~~~~~~~~~
 
+- Bug where groupby.agg (first, last, min, etc...) returns incorrect results for uint64 columns. (:issue:`26310`)
```

Comment:
move to grouping use full references (see other issues on how to do this)

Reply:
done